crates/api_crud/src/comment/create.rs

```diff
@@ -164,10 +164,12 @@ pub async fn create_comment(
   )
   .await?;
 
-  // If its a reply, mark the parent as read
+  // If its a reply to us, mark the parent as read
   if let Some(parent) = parent_opt {
+    let person_id = local_user_view.person.id;
```

Comment:
If it's a reply to the local user, wouldn't the parent belong to the local user?

Reply:
I made a more detailed comment.